pkg/storage/raft_log_queue.go

```diff
@@ -180,7 +180,9 @@ func computeTruncatableIndex(
 // getQuorumIndex returns the index which a quorum of the nodes have
 // committed. The pendingSnapshotIndex indicates the index of a pending
 // snapshot which is considered part of the Raft group even though it hasn't
-// been added yet.
+// been added yet. Note that getQuorumIndex may return 0 if the progress map
+// doesn't contain information for a sufficient number of followers (e.g. the
+// local replica has only recently become the leader).
```

Comment:
It may be worth mentioning that using `raftStatus.Commit` is more appropriate in some circumstances as a result.